tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/statements/arrays/SealedArrayTest.java

```diff
@@ -275,22 +275,22 @@ public void testUnionAndMatchNoSealedArrayStatement() {
                 "matched float array", "Couldn't match unsealed array type");
     }
 
-    @Test(enabled = false)
+    @Test()
     public void testNegativeSealedArrays() {
         Assert.assertEquals(resultNegative.getErrorCount(), 24);
         int i = 0;
-        BAssertUtil.validateError(resultNegative, i++, "array index out of range: index: '5', size: '5'", 19, 30);
-        BAssertUtil.validateError(resultNegative, i++, "array index out of range: index: '5', size: '5'", 25, 33);
+        BAssertUtil.validateError(resultNegative, i++, "list index out of range: index: '5'", 19, 18);
```

Comment:
The column number should be the original value. 

Can you please change the pos at https://github.com/ballerina-platform/ballerina-lang/blob/master/compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/TypeChecker.java#L4088 from `indexBasedAccessExpr.pos` to `indexBasedAccessExpr.indexExpr.pos` and check? If this breaks other tests, let's create an issue and go ahead and merge this.

Reply:
Made the required changes.